### PR TITLE
docs: add note about chainsaw

### PR DESCRIPTION
**Pull request by eddycharly**
**Description of the change:**

Add note about chainsaw in the docs.

If you need more infos :point_down:
- GitHub: https://github.com/kyverno/chainsaw
- Docs: https://kyverno.github.io/chainsaw
- Slack: https://kubernetes.slack.com/archives/C067LUFL43U

**Motivation for the change:**

We presented chainsaw to the community meeting last week and received good feedback.
It would be awesome to have it listed in the docs.

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Comment by camilamacedo86** [comment]
Following the discussion we had via Slack, I wanted to share some thoughts on the PR.

The operator-sdk includes a feature called scorecard, which utilizes kuttl. This is the reason kuttl is mentioned in the documentation. However, chainsaw is not included or shipped with the project. Therefore, I'm not sure if it's appropriate to include it in the documentation.

Including chainsaw might lead the operator-sdk maintainers into a situation where they have to answer questions and provide support for a tool that is not a direct part of the project. It can be challenging for maintainers to support third-party tools and integrations.

**I am no longer an active maintainer of the SDK project, so I'll leave the decision to those currently maintaining the project to determine what best fits their strategy.**

**Comment by eddycharly** [comment]
Thanks for chiming in @camilamacedo86 !

Operator SDK can support chainsaw scorecard test, here is the PR https://github.com/operator-framework/operator-sdk/pull/6676 😄 

I'm fine with answering chainsaw related questions if any.

**Comment by eddycharly** [comment]
@jberkhahn do you have any suggestion after the yesterday meeting ? what is the next step ?

**Review by varshaprasad96 — APPROVED** [review]
This looks good to me. As a scaffolding tool we are listing the possible options and redirecting user to the external documentation. 

Thanks for the PR @eddycharly!
/lgtm

**Comment by eddycharly** [comment]
Thanks @varshaprasad96 😍

Do I need to fix something ?

File: website/content/en/docs/building-operators/golang/testing.md [hunk 1]
```diff
@@ -59,3 +61,5 @@ To implement application-specific tests, the SDK's test harness, [scorecard][sco
 [go-legacy-shell]: https://github.com/operator-framework/operator-sdk/blob/v1.0.0/hack/tests/e2e-go.sh
 [helm-legacy-shell]: https://github.com/operator-framework/operator-sdk/blob/v1.0.0/hack/tests/e2e-helm.sh
 [ansible-legacy-shell]: https://github.com/operator-framework/operator-sdk/blob/v1.0.0/hack/tests/e2e-ansible.sh
+[chainsaw]: https://kyverno.github.io/chainsaw
```
Comment (eddycharly) [inline website/content/en/docs/building-operators/golang/testing.md:64]: Maybe i need to set it to https://kyverno.github.io/chainsaw/latest/ ?

Comment (eddycharly) [inline website/content/en/docs/building-operators/golang/testing.md:64]:
```suggestion
[chainsaw]: https://kyverno.github.io/chainsaw/latest/
```

**Comment by eddycharly** [comment]
@varshaprasad96 i lost your LGTM 🙏

**Comment by varshaprasad96** [comment]
Going to go ahead and merge this, as it has been open for review for a few days! If folks have any issues/further discussions w.r.t to this PR, please feel free to create issues :)

**Pull request merged**